src/deluge/gui/views/instrument_clip_view.cpp

```diff
@@ -2161,59 +2161,64 @@ void InstrumentClipView::adjustProbability(int32_t offset) {
 
 		// Decide the probability, based on the existing probability of the leftmost note
 		probabilityValue = editPadPresses[leftMostIndex].intendedProbability & 127;
-		probabilityValue += offset;
```

Comment:
the row above this modifies the probability value

Reply:
We need to get the current probability and I think that line is the way of getting it. Then modified by the second line